client/public/src/components/app.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+
+import Details from './details.jsx';
+import Options from './options.jsx';
+
+const App = () => {
+  const [productId, setProductId] = useState(1);
```

Comment:
I wonder if you can simplify this a little and remove the `productId` and `storeId` states? It doesn't look like these change anywhere--could you parse them out of the `product` and `store` variables when needed?

Reply:
These are the selected product, and selected store, but I changed the name to make it shorter. I figured if I were to make it possible to switch stores or product, I would have to keep them in a state.